salalib/shapemap.cpp

```diff
@@ -2873,7 +2937,7 @@ bool ShapeMap::importData(const depthmapX::Table &data, std::vector<int> indices
 
                     if(colcodes.size() >= 32) {
                         for (size_t j = 0; j < column.second.size(); j++) {
-                            m_attributes.setValue(indices[j], colIndex, -1.0f);
+                            m_attributes.setValue(m_attributes.getRowid(indices[j]), colIndex, -1.0f);

```

Comment:
Not sure I understand this change? Isn't set value index based rather than key based?

Reply:
Yes, but we are not carrying indices anymore but shape_refs (that's what the various make* functions above return), so now we need to find the index of that ref in the attributes table. I'll just rename the vectors to indicate that they carry shaperefs